### DISCUSSION TITLE
Unable to upload H5p and HTML content

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Knowlg/Community#205] [Bugs]
Hi Team,
Recently we upgraded to 5.1.0 after the upgrade we are not able to upload H5p and HTML content we are getting the below error mentioned in the screenshot.
![Screenshot from 2024-04-23 13-16-34](https://github.com/Sunbird-Knowlg/Community/assets/128453924/de666eec-ab01-4ea9-a069-1b4f71adc629)

As per out understanding content is getting uploaded to a container successfully that is **sunbird-content-dev** container based on the network tab we noticed it is searching for the content in the container called **Content** and giving 404 error.
The new **sunbird-content-dev** container is created based on this variable added as per 5.1.0  **cloud_storage_content_bucketname: "sunbird-content-{{env}}"**
Kindly need help on this.

cc: @raghupathiguduri

**Comment by pallakartheekreddy** [comment]
@Venkatesh-Dayani23 Thank you for your patience. Does this issue still exist? Can we get into a call to get additional information about this issue.

**Reply by Venkatesh-Dayani23** [reply]
Hi @pallakartheekreddy 
The issue still exists. Kindly let us know when can we connect on this issue.

**Reply by Venkatesh-Dayani23** [reply]
Hi @pallakartheekreddy 
Can you give any update on this please ?

**Reply by pallakartheekreddy** [reply]
@Venkatesh-Dayani23 Can we connect today 12PM?

**Reply by Venkatesh-Dayani23** [reply]
Hi @pallakartheekreddy 
Thank you for your support The issue is fixed.

**Reply by pallakartheekreddy** [reply]
@Venkatesh-Dayani23 Can you provide details how it got fixed? It will help other adopters..
